hivemind/optim/grad_averager.py

```diff
@@ -224,3 +230,10 @@ def use_averaged_gradients(self):
     def notify_used_averaged_gradients(self):
         """Notify averager that the results of a previous averaging round are accounted for"""
         self._new_averaged_grads = False
+
+    @classmethod
+    def get_factory(cls, **kwargs1) -> GradientAveragerFactory:
```

Comment:
Nitpicking: consider functools.partial

Reply:
Yes, the factories look overcomplicated to me, please try to use `partial()` instead (and remove the factories) if possible